app/javascript/components/analytics/widgets/MonitoreesByEventDate.js

```diff
@@ -51,9 +51,9 @@ class MonitoreesByEventDate extends React.Component {
     } else if (timeRes === 'Month') {
       dateRangeInQuestion = 'Last Exposure Month';
     }
-    DATES_OF_INTEREST = _.uniq(this.props.stats.monitoree_counts.filter(x => x.category_type === dateRangeInQuestion).map(x => x.category))
-      .sort()
-      .slice(0, 14);
+    DATES_OF_INTEREST = _.uniq(
```

Comment:
Can you describe what this was fixing exactly? I want to make sure I'm following (and testing) correctly.

Reply:
Before, I had only been allowing 14 items to be shown. Now, we will show as far back as the backend has data. Given that everyone is `active: true`, it wont go back very far. When/if we add the ability to toggle between Active and All, this "trim" functionality may come back.